content/en/docs/instrumentation/php/logging.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Logging
```

Comment:
General question to @svrnm @cartermp @austinlparker et al.: should the title here be **Logs** or **Logging**? (Whatever we choose, we'll probably want to consistently apply across languages.)

Reply:
I think Logging is better than Logs for now. Since it's in the context of _doing logging_ with an SDK.

Reply:
I vote for "Logging"

Reply:
Thanks for chiming in! That works for me.